dkpro-core-io-brat-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/io/brat/BratWriter.java

```diff
@@ -610,13 +615,42 @@ private void writeRelationAnnotation(BratAnnotationDocument aDoc, FeatureStructu
         }
     }
 
+
+    
+    private BratTextAnnotation splitNewline(AnnotationFS aFS)
+    {
+
+        Pattern p = Pattern.compile("(.+?)(?:\\R|$)+", Pattern.DOTALL);
```

Comment:
Patterns can be pre-compiled and stored a static final constants in the class. Also please provide a comment what the purpose of this pattern is.